tests/test_callprofiler.py

```diff
@@ -192,3 +192,95 @@ def foo():
 
         modmesh.call_profiler.reset()
         foo()
+
```

Comment:
1. I rebase and squash another developing branch `issue343`.
2. At the localhost, I run `git checkout -b issue343_pr` under the master branch to revert the latest `solvcon
/modmesh`.
3. I merge the developing branch `issue343` to this branch `issue343_pr`.
4. I forcely push the commit to `origin/issue343_pr` to modify the history.

I'm not sure whether I'm wrong to reduce and delete the unnecessary commit.

Reply:
Your history contains commits from @tigercosmos @j8xixo12 @YenPeiChen07.  That does not look right.  Please rebase against `master`, remove commits not belonging to you, and squash small commits.

Reply:
> Your history contains commits from @tigercosmos @j8xixo12 @YenPeiChen07. That does not look right. Please rebase against `master`, remove commits not belonging to you, and squash small commits.

Fix.

Reply:
It looks good now.  Thanks.